xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererDRMPRIMEGLES.cpp

```diff
@@ -188,3 +222,32 @@ bool CRendererDRMPRIMEGLES::RenderHook(int index)
 
   return true;
 }
+
+void CRendererDRMPRIMEGLES::AfterRenderHook(int index)
+{
+  m_fences[index]->CreateFence();
+}
+
+bool CRendererDRMPRIMEGLES::Supports(ERENDERFEATURE feature)
+{
+  if (feature == RENDERFEATURE_STRETCH ||
```

Comment:
same as above

these kind of functions (SupportsXXX) also often use switch/case constructs